### PR TITLE
docs: add a doc about using dragonfly with harbor registry

### PR DESCRIPTION
**Pull request by Starnop**
Signed-off-by: Starnop <starnopg@gmail.com>

### Ⅱ. Does this pull request fix one issue?

ref #1094

### CODE REVIEW

**Comment by yeya24** [comment]
Is it better to put this document into docs/ecosystem folder?

File: docs/ecosystem/Harbor-with-Dragonfly.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+If the output of command above has content like
+
+```
+2019-03-29 15:49:53.913 INFO sign:96027-1553845785.119 : downloading piece:{"taskID":"00a0503ea12457638ebbef5d0bfae51f9e8e0a0a349312c211f26f53beb93cdc","superNode":"127.0.0.1","dstCid":"127.0.0.1-95953-1553845720.488","range":"67108864-71303167","result":503,"status":701,"pieceSize":4194304,"pieceNum":16}
+```
+
+that means that the image download is done by Dragonfly.
+
+If you need to ensure that if the image is transferred through other peer nodes, you can execute the following command:
```
Comment (yunfeiyanggzq) [inline docs/ecosystem/Harbor-with-Dragonfly.md]: `if` -> `whether`?

**Comment by allencloud** [comment]
LGTM

**Comment by allencloud** [comment]
I would like to invite @lowzj and @yeya24 to review this doc again.

However, this doc mostly emphasizes on how to pull images from Harbor with Dragonfly. While quite a few people care about how to preheat images from Harbor to Dragonfly's supernode.

Am I right? I think we should add two parts above in doc, maybe not in the same document. @Starnop

**Comment by Starnop** [comment]
> I would like to invite @lowzj and @yeya24 to review this doc again.
> 
> However, this doc mostly emphasizes on how to pull images from Harbor with Dragonfly. While quite a few people care about how to preheat images from Harbor to Dragonfly's supernode.
> 
> Am I right? I think we should add two parts above in doc, maybe not in the same document. @Starnop

I will add it with a new PR.

**Pull request merged**